src/main/sensors/gyro_init.c

```diff
@@ -182,7 +182,7 @@ static bool gyroInitLowpassFilterLpf(int slot, int type, uint16_t lpfHz, uint32_
     *lowpassFilterApplyFn = nullFilterApply;
 
     // If lowpass cutoff has been specified and is less than the Nyquist frequency
-    if (lpfHz && lpfHz <= gyroFrequencyNyquist) {
+    if (lpfHz) {
```

Comment:
If we're not going to do the nyquist check here the comment should be changed too.

Reply:
Yes, makes sense.